docs/src/contribute/source-code.md

```diff
@@ -1,11 +1,10 @@
 ---
-title: Source Code
+title: Get the Source Code
 eleventyNavigation:
-    key: getting the source code
-    parent: developer guide
-    title: Getting the Source Code
-    order: 1
-
+    key: get the source code
+    parent: contribute to eslint
+    title: Get the Source Code
+    order: 11
```

Comment:
While this is still in the sidebar, can we keep it right before "Set Up a Development Environment"?

Reply:
i can move it up. i just put at the bottom so we wouldn't have to change TOC order for all the elements when the change happens. the PR to remove this page is already in draft https://github.com/eslint/eslint/pull/16780.

Reply:
moved up in 583c6da8d